tests/Integration/Behaviour/Features/Scenario/Order/order_from_bo.feature

```diff
@@ -698,3 +698,24 @@ Feature: Order from Back Office (BO)
     And the first invoice from order "bo_order1" should have following shipping tax details:
       | total_tax_excl | rate | total_amount |
       | 7              | 6.00 | 0.42         |
+
+  Scenario: View order passed with a now deleted language
+    Given language "Spanish" with locale "es-ES" exists
+    And language with iso code "es" is the default one
+    When I create an empty cart "dummy_cart-ES" for customer "testCustomer"
+    And I select "US" address as delivery and invoice address for customer "testCustomer" in cart "dummy_cart-ES"
+    And I add 1 product "Mug The best is yet to come" to the cart "dummy_cart-ES"
+    And I add order "bo_order-ES" with the following details:
+      | cart                | dummy_cart-ES               |
+      | message             |                             |
+      | payment module name | dummy_payment               |
+      | status              | Awaiting bank wire payment  |
+    And I change order "bo_order-ES" shipping address to "test-address"
+    Then order "bo_order-ES" shipping address should be "test-address"
+    And order "bo_order-ES" preview shipping address should have the following details:
+      | country | United States |
```

Comment:
What I don't understand is that it's actually never displayed in spanish? How can you tell the `en` language is not used all the time?

Reply:
The purpose of this test is to see if you can display an order created with a currently deleted language (and I assure you, it broke before the modifications ^^), maybe a different test should be written (if not already existing) to check that when you change the default language, it is actually used when creating an order?